src/devices/Nrf24l01/Nrf24l01.cs

```diff
@@ -200,23 +200,25 @@ private void Irq_ValueChanged(object sender, PinValueChangedEventArgs args)
         }
 
         #region private and internal
+
         /// <summary>
         /// Initialize
         /// </summary>
         private void Initialize(PinNumberingScheme pinNumberingScheme, OutputPower outputPower, DataRate dataRate, byte channel)
         {
+            // open pins
             _gpio = new GpioController(pinNumberingScheme);
             _gpio.OpenPin(_ce, PinMode.Output);
             _gpio.OpenPin(_irq, PinMode.Input);
             _gpio.RegisterCallbackForPinValueChangedEvent(_irq, PinEventTypes.Falling, Irq_ValueChanged);
 
-            Thread.Sleep(50);
+            Thread.Sleep(10);       // wait for some time
```

Comment:
why do you need to wait? perhaps this is something we can fix on our side?

Reply:
No, I just think we need to give the hardware a little reaction time to make sure it works correctly.😅